Zentra_download.py

```diff
@@ -1,17 +1,48 @@
-# Python script to download Soil data from Zentra Cloud
-# Sujan Pal, ANL EVS - 2022
-
-from zentra.api import *; import os; import pandas as pd; import time
-os.chdir('/home/spal/notebooks/CROCUS/')
-token = ZentraToken(username=os.getenv("zentra_un"),password=os.getenv("zentra_pw"))
-Device_SN = ["z6-04628","z6-04625","z6-04619","z6-04629","z6-04641","z6-04638","z6-04624","z6-04627","z6-04620","z6-04621","z6-04618"]
+Downloads Soil Data
+Sujan Pal and Jack Colwell
+Data from: loggers located at ATMOS and data collected from Zentra website
+Imports
+from zentra.api import *
+import os 
+import pandas as pd
+import time
+Downloads Data from the Loggers at the Zentra Website
+os.chdir(r'C:\Users\colwe\OneDrive\Desktop\Argonne\ATMOS484\ATMOS484-master\input\TEROS')
+​
+#This token allows one to login to the Zentra website
+token = ZentraToken(username='jcolwell@anl.gov',password='!Jc161616!')

```

Comment:
It is not a good practice to make username and password public. username=os.getenv("zentra_un"),password=os.getenv("zentra_pw") is advised.